service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -90,26 +83,4 @@ void aboutToSubmitShouldPopulateListServedCaseManagementOrdersWhenSendsToAllPart
                     .build())
                 .build()));
     }
-
-    private CallbackRequest buildCallbackRequest(Map<String, Object> data) {
-        return CallbackRequest.builder()
-            .eventId(ACTION_CASE_MANAGEMENT_ORDER.getId())
```

Comment:
how does it work with removal of eventId? by coincidence now? or was the scenario not checked?

Reply:
good question, checked and it looks that service is ding 
if (DRAFT_CASE_MANAGEMENT_ORDER.getId().equals(eventId)) {
            progressDraftCaseManagementOrder(caseDetails, caseData.getCaseManagementOrder());
        } else {
            progressActionCaseManagementOrder(caseDetails, caseData);
        }
so it works same without eventId,
But yes, it is not right and eventId should be pass by test